src/cadenza.js

```diff
@@ -118,9 +118,11 @@ globalThis.cadenza = Object.assign(
  * @typedef {'m'|'km'} LengthUnit
  */
 
+/** @typedef {number[]} Coordinate - A tuple with an x and y value // [number, number] */
```

Comment:
The type should be `[number, number]` instead of `number[]`. Then you also don't need to mention that in the description.